src/makeCalculator.js

```diff
@@ -5,6 +5,35 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(num) {
+      result += num;
+    },
+    subtract(num) {
+      result -= num;
+    },
+    multiply(num) {
+      result *= num;
+    },
+    divide(num) {
+      result /= num;
+    },
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics